test/test_tree_1d_mhd.jl

```diff
@@ -64,6 +64,7 @@ EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "tree_1
   end
 
   @trixi_testset "elixir_mhd_shu_osher_shock_tube.jl with flipped shock direction" begin
+    trixi_include(joinpath(EXAMPLES_DIR, "elixir_mhd_shu_osher_shock_tube.jl"), tspan=(0.0, 0.0))
```

Comment:
What is this for?

Reply:
Getting the definition of `initial_condition_shu_osher_shock_tube_flipped` used below

Reply:
Uagh, this is really ugly. Maybe at least add a comment?
```suggestion
    # Include this elixir to make `initial_condition_shu_osher_shock_tube_flipped` available, which is used below
    trixi_include(joinpath(EXAMPLES_DIR, "elixir_mhd_shu_osher_shock_tube.jl"), tspan=(0.0, 0.0))
```